assets/performanceprofile/scripts/low-latency-hooks.sh

```diff
@@ -1,5 +1,22 @@
 #!/usr/bin/env bash
 
+function rps_mask_set() {
+    local pid=$1
+    local mask=$2
+    local rps_files
+
+    rps_files=$(find /proc/"${pid}"/root/sys/devices/virtual -type f -name rps_cpus)
```

Comment:
My main concern here is that this path is not modified for other containers (once the RPS mask of the network ns already set for other container in which share the same ns) and still represents the old rps mask.

So this function will always return `false` and the hook will run for each container in the pod and not only once as intended.